.travis.yml

```diff
@@ -19,7 +19,7 @@ env:
   - secure: Br1GCNbRB0Q3lgUW1xq9ULFd+wQdE6YdHYqDlt51007s88KBsY5J/yhvfGQss4EDodxCTZQRPfvDqngdUKjFVARjSezv91vRBIuE5JJfJGLuYTmRmLlUj0I7P5tdJPgkPBbIBy7Nwf8ztjS1qvkib1/Yi+DEvC8CfQ/34zvTpyiAmYuOrON7ThCm+20r3boVEwpKZ9xUjTPU7rkaDPEv8ksQwek/rie9VCGGOdyv3sOascICTCsyZjYCYitSQ2oqSXfRBN3K6+4Pb8UufeE1hLvoDOBrbJIikSTk+1o6+gXFd2ME0IHvgS2pR2vNkoFCM3Wgg1dT4XNMKk1MZopDBQNMTX0DeNbw6wkfr/bIGzzrCGf7t2cc7ziK45Neup1vDldfuwWrvV3IWsIaAeYuLmQ8r6CUSd9Th/x+keg+Lt1r1OODz5pTV+lheNErrFs3PJpJwsFFc2FkuCCTuwSDxf3UveWIgmHKX6aZWr6TwMU7xpclWftb0bJKxn9+sS8zsXNM/V2Wj90YgpMJpqIXIGN3/8+bJzGXn/5+pz0WfzAczHfkyu/PyCmcleRxody3pE/YoiXrM6kIUNOGDqp6pSkZxYt/DtRRvr/G67oyz2aTKML0wpvThezxiWcGv0tKyrYAeL+7ES9dOYJM0CwNkgVt3/UNR17pAgDYDb3FuBs=
 script:
 - 'if [[ "$TRAVIS_SECURE_ENV_VARS" = "true" ]]; then bash ./travis/jekyll_build.sh; fi'
-- 'if [[ "$TRAVIS_SECURE_ENV_VARS" != "true" ]]; then bundle exec jekyll build; fi'
+- 'if [[ "$TRAVIS_SECURE_ENV_VARS" != "true" ]]; then travis_wait bundle exec jekyll build; fi'
```

Comment:
Nice find on this function!  But looking at the documentation, the default is a 20 min limit.  Do we want to make it longer?  `travis_wait 40` for example?

Reply:
@blueandgold Perhaps we start with 20 minutes and if we need to, we'll increase it later?

Reply:
@blueandgold @kevensen I found we were hitting the 10 min no output timeout which is the specific issue I am trying to address. I don't think we should extend beyond the default. If the website build time gets to 20 or 30 mins we have a different problem to tackle which is pruning the number of versions we build.